### ISSUE TITLE
UNKNOWN ERROR: Too many object types to load: 231 on Insert to TopicAssignment object

### ISSUE THREAD

**Issue by ScottInLondon** [ryanbrainard/forceworkbench#659]
I tried to insert 55 TopicAssignment objects from a CSV. On the field-mapping screen I got this error message - presumably because TopicAssignment.EntityId points to a small number of objects (those enabled for the Topics on Objects feature) but also to FeedItems, of which the number is large.

![screenshot 2015-07-27 at 12 39 51](https://cloud.githubusercontent.com/assets/13520560/8905252/97796558-345c-11e5-97ad-882f9744c9f1.png)

**Comment by sxnahm** [comment]
I had a similar issue, and was told to try "disabling "Smart Lookup" in Workbench Settings." That allowed me to get past this error.

**Comment by ryanbrainard** [comment]
Correct, that’s the fix. We should probably add something to the error message to make it clearer.
